ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/helpers/OfferingToStateMapper.kt

```diff
@@ -24,14 +25,18 @@ internal fun Offering.validatedPaywall(
             PaywallData.createDefault(packages = availablePackages, currentColorScheme),
             PaywallData.defaultTemplate,
             PaywallValidationError.MissingPaywall,
-        )
+        ).also {
+            Logger.d(PaywallValidationErrorStrings.MISSING_PAYWALL.format(this.identifier))
```

Comment:
Hmm I wonder if this should be a warning... I think displaying the default paywall shouldn't be a fallback but shouldn't be relied on normally... Not sure if there will be use cases where people don't want to configure anything in the dashboard though.

Same for the others, I might do them warnings.

Reply:
This is a good point. In iOS we display them in the view, which is a pretty big warning I would say. What do you think @NachoSoto